hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -174,8 +174,8 @@ def queries(query_group,
         if res['retcode'] == 0:
             query_ret['data'] = json.loads(res['stdout'])
         else:
-            query_ret['result'] = False
-            query_ret['error'] = res['stderr']
+            queury_ret['result'] = False
+            queury_ret['error'] = res['stderr']
```

Comment:
I think these misspellings got added when you rebased/merged with upstream. Can you fix it back to `query_ret` on both lines?